packages/transport/src/create.ts

```diff
@@ -185,7 +185,8 @@ export const createChannelTransport = ({
         return streamImpl as MethodImpl<typeof method>;
       }
       default: {
-        const noImpl: unknown = () => Promise.reject(`${MethodKind[method.kind]} unimplemented`);
+        const noImpl: unknown = () =>
+          Promise.reject(new Error(`${MethodKind[method.kind]} unimplemented`));
```

Comment:
This is due to that same linting issue where `reject()` should take an `Error` object. Again, I'm not sure how to test this.